config/owasp/suppressions.xml

```diff
@@ -29,21 +29,21 @@
     <vulnerabilityName>CVE-2023-41080</vulnerabilityName>
   </suppress>
 
-  <suppress until="2023-09-14">
+  <suppress until="2023-10-14">
     <notes>Looks like a new security issue which involves DOS attacks</notes>
     <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-databind@.*$</packageUrl>
     <vulnerabilityName>CVE-2023-35116</vulnerabilityName>
   </suppress>
 
-  <suppress until="2023-09-14">
+  <suppress until="2023-10-14">
     <notes>The exploitation depends on the structure of the target LDAP directory, as well as what kind of errors are
       exposed to the user.
     </notes>
     <packageUrl regex="true">pkg:maven/org.bouncycastle/bcprov-jdk15on@1.69</packageUrl>
     <vulnerabilityName>CVE-2023-33201</vulnerabilityName>
   </suppress>
```

Comment:
Di we still need this supression, looks like the suppression only applies to v1.69 but looks like we're using v1.73 now:
```
$ ./gradlew dependencyInsight --dependency bcprov

> Configure project :
FLYWAY url=null
FLYWAY schemas=[darts]

> Task :dependencyInsight
org.bouncycastle:bcprov-jdk18on:1.73
  Variant compile:
    | Attribute Name                 | Provided | Requested    |
    |--------------------------------|----------|--------------|
    | org.gradle.status              | release  |              |
    | org.gradle.category            | library  | library      |
    | org.gradle.libraryelements     | jar      | classes      |
    | org.gradle.usage               | java-api | java-api     |
    | org.gradle.dependency.bundling |          | external     |
    | org.gradle.jvm.environment     |          | standard-jvm |
    | org.gradle.jvm.version         |          | 17           |

org.bouncycastle:bcprov-jdk18on:1.73
+--- org.bouncycastle:bcpkix-jdk18on:1.73
|    \--- org.springframework.security:spring-security-rsa:1.0.12.RELEASE
|         \--- org.springframework.cloud:spring-cloud-starter:4.0.4
|              \--- org.springframework.cloud:spring-cloud-starter-openfeign:4.0.4
|                   \--- compileClasspath
\--- org.bouncycastle:bcutil-jdk18on:1.73
     \--- org.bouncycastle:bcpkix-jdk18on:1.73 (*)
```

Reply:
ah ok. I'll check and remove if necessary